include/uapi/sound/sof/tokens.h

```diff
@@ -114,4 +114,8 @@
 #define SOF_TKN_IMX_ESAI_FIRST_TOKEN		1100
 /* TODO: Add ESAI tokens */
 
+/* Stream */
```

Comment:
fix commit title
ASoC: SOF: token: add token for PCM compatible with D0i3 substate